src/components/space/spaceNotification/spaceNotification.tsx

```diff
@@ -0,0 +1,118 @@
+// React + Web3 Essentials
+import React from 'react';
+
+// External Packages
+import styled from 'styled-components';
+
+// Internal Components
+import { ButtonV2, ImageV2, ItemHV2, ItemVV2, SpanV2 } from 'components/reusables/SharedStylingV2';
+import { spaces } from 'services/space/spaceList';
+import { shortenText } from 'helpers/UtilityHelper';
+import { ReactComponent as Space } from 'assets/space/space.svg';
+
+// Internal Configs
+import { device } from 'config/Globals';
+
+const SpaceNotification = () => {
+  const selectedSpace = spaces[0];
+
+  const joinSpace = () => {};
+
+  return (
+    <NotificationContainer>
+      <NotificationHeader>
+        <ItemHV2 justifyContent="flex-start">
+          <ItemVV2
+            width="32px"
+            height="32px"
+            maxWidth="32px"
+            borderRadius="50%"
+            overflow="hidden"
+            margin="0px 5px 0px 0px"
+          >
+            <ImageV2
+              height="100%"
+              objectFit="cover"
+              src={selectedSpace?.spaceImage}
+              alt=" Image"
+            />
+          </ItemVV2>
+          <SpanV2
+            color="#F5F5F5"
+            fontSize="17px"
+            fontWeight="500"
+            margin="0px 5px 0px 0px"
+          >
+            {shortenText(selectedSpace?.spaceCreator, 6)}
+          </SpanV2>
+          <SpanV2
+            padding="4px 8px"
+            borderRadius="8px"
+            background="rgba(255, 255, 255, 0.2)"
+            fontSize="12px"
+            fontWeight="500"
+            color="#FFFFFF"
+          >
+            Host
+          </SpanV2>
+        </ItemHV2>
+        <ItemHV2 justifyContent="flex-end">
+          <Space />
+          <SpanV2
+            fontSize="14px"
+            fontWeight="500"
+            color="#FFFFFF"
+            margin="0px 0px 0px 3px"
+          >
+            Live
+          </SpanV2>
+        </ItemHV2>
+      </NotificationHeader>
+      <SpanV2
+        fontSize="20px"
+        fontWeight="500"
+        color="#F5F5F5"
+        alignSelf="flex-start"
+        margin="0px 0px 12px 0px"
+      >
+        {`${shortenText(selectedSpace?.spaceCreator, 6)}'s Space`}
+      </SpanV2>
+      <JoinButton onClick={() => joinSpace()}>Join this space</JoinButton>
+    </NotificationContainer>
+  );
+};
+
+const NotificationHeader = styled(ItemHV2)`
+  align-items: center;
+  justify-content: space-between;
+  max-height: 32px;
+  width: 100%;
+  margin-bottom: 14px;
+`;
+
+const NotificationContainer = styled(ItemVV2)`
+  position: fixed;
+  right: 51px;
+  bottom: 84px;
```

Comment:
<img width="589" alt="Screenshot 2023-04-19 at 2 37 41 PM" src="https://user-images.githubusercontent.com/77395788/233027238-ddf1ec3b-d473-4d5a-bddd-a3d5d8b435c9.png">
 
 The distance of snackbar should be more from the bottom as it should not overlap the profile section